src/pcm/AudioFormat.cxx

```diff
@@ -22,15 +22,46 @@
 
 #include <assert.h>
 #include <stdio.h>
+#include <map>
+
+/**
+ * If source is based on 44.1k get nearest valid 44.1k variant from the target sample rate.
+ * For exmple if the source is 44.1k and the target is 96k it will return 88.2k
+ */
+unsigned
```

Comment:
Why have this global function's prototype inside the C++ file? Why is this even global?
I don't understand the documentation.